### PR TITLE
Remove borken parts of Makefile and repair manpage creation, clarify license

### LINKED ISSUE #85: Makefile is broken
The makefile is no longer working, since there is no longer a codespell.py

### PR DESCRIPTION
**Pull request by cinemast**
Actually fixes #85 by removing the broken parts and fixing the useful parts in the Makefile.

### CODE REVIEW

**Comment by cinemast** [comment]
If this gets merged, a new release would also be highly appreciated, the last tag is from 6 months ago.

**Comment by cinemast** [comment]
I don't mean to rush, but is there a chance to merge this within the next 2 weeks?

**Comment by Eric89GXL** [comment]
LGTM, I'll merge in a few days unless someone else wants to have a look

File: Makefile [hunk 1]
```diff
@@ -33,37 +17,8 @@ check-dictionary:
...
-
-tar-sync: codespell-$(VERSION).tar.xz codespell-$(VERSION).tar.xz.asc
-	github-release release --repo codespell --tag v$(VERSION) --name v$(VERSION)
-	github-release upload  --repo codespell --tag v$(VERSION) \
-		--name codespell-$(VERSION).tar.xz \
-		--file codespell-$(VERSION).tar.xz
-	github-release upload  --repo codespell --tag v$(VERSION) \
-		--name codespell-$(VERSION).tar.xz.asc \
-		--file codespell-$(VERSION).tar.xz.asc
-
```
Comment (lucasdemarchi) [inline Makefile]: why are you removing the parts from the makefile that are used to make a release?  How are we going to do releases now ?  We should certainly tag the version in the repository, even if we don't upload tarballs anymore.

Comment (Eric89GXL) [inline Makefile]: IIRC I just tagged a release on the GitHub GUI, then did `python setup.py sdist upload`

Comment (lucasdemarchi) [inline Makefile]: ok, thanks for clarifying.

**Comment by cinemast** [comment]
Just wanted to ping on that. Is there a problem with this PR?

**Comment by Eric89GXL** [comment]
@lucasdemarchi okay with you?

**Comment by lucasdemarchi** [comment]
> @lucasdemarchi okay with you?

Yep. Let's merge this and prepare a release.

**Comment by Eric89GXL** [comment]
Thanks @cinemast

**Pull request merged**